make/workflows.mk

```diff
@@ -0,0 +1,4 @@
+.PHONY: workflow-labeler-yaml-update
+workflow-labeler-yaml-update: ## Updates .github/service-labeler.yaml for use with labeler GH action
+workflow-labeler-yaml-update: ; $(info $(M) updating .github/service-labeler.yaml with latest services)
+	ls -d services/* | awk 'NR > 1' | sed p | awk 'NR % 2 { print $$0 ":" } !(NR % 2) {print "  - " $$0 "/**";}' > .github/service-labeler.yaml
```

Comment:
fancy :exploding_head:

Reply:
would be helpful if you can add a comment as to whats happening here in the code to make future edits easy :see_no_evil:

Reply:
added in https://github.com/mesosphere/kommander-applications/pull/681/commits/59471402328e269e62b6cfe362dde5ae8c29a987